repository.py

```diff
@@ -1,15 +1,22 @@
 import pymongo
 from pymongo import MongoClient
 import json
+import os
 
 def get_connection():
     return MongoClient('localhost', 27017)['gitnomic']
-  
+
 def get_config():
+  if os.environ.get('LOCAL_DEV'):
+    return {
+        'git_repo': 'fakeRepo',
+        'git_login': 'fakeLogin',
+        'git_password': 'fakePassword'
```

Comment:
Oh no!
Theres no way to add a 'db_setup' config.

Without that, the application will run the init_db method if you ever actually restart it.

try docker-compose down then docker-compose up and you will see that mongodb throws a duplicate key error when it tries to perform first time setup again.